.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: mbgl/ci:r4-linux-clang-3.9
+    working_directory: ~/mapbox-gl-js
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-mapbox-gl-js-cache
+      - run:
+          name: install dependencies
+          command: |
+            bash ./.circleci/dependencies.sh
+      - save_cache:
+          key: v1-mapbox-gl-js-cache
+          paths:
+            - '~/.yarn'
+            - '.eslintcache'
+            - 'node_modules'
+      - run:
+          name: run tests
+          command: |
+            bash ./.circleci/test.sh
+      - store_artifacts:
+          path: "test/integration/render-tests/index.html"
+      - store_artifacts:
+          path: "test/integration/query-tests/index.html"
+      - run:
+          name: deploy if tagged
+          when: on_success
+          command: |
+            if [[ $CIRCLE_TAG =~ v[0-9]+\.[0-9]+\.[0-9]+(\-dev)? ]]; then
```

Comment:
It looks like something is off about this conditional; https://circleci.com/gh/mapbox/mapbox-gl-js/8468 ran `deploy.sh` but it wasn't a tagged build.